packages/graphics/cairo/package.mk

```diff
@@ -3,11 +3,12 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="cairo"
-PKG_VERSION="1.17.4"
-PKG_SHA256="74b24c1ed436bbe87499179a3b27c43f4143b8676d8ad237a6fa787401959705"
+PKG_VERSION="1.17.6"
```

Comment:
1.17.6 is just a snapshot and not a release (awesome version numbering .... wtf)
https://gitlab.freedesktop.org/cairo/cairo/-/tags

Reply:
not very consistent :-(